graphql2/graphqlapp/destinationtypes.go

```diff
@@ -243,13 +246,14 @@ func (q *Query) DestinationTypes(ctx context.Context) ([]graphql2.DestinationTyp
 			}},
 		},
 		{
-			Type:                destWebhook,
-			Name:                "Webhook",
-			Enabled:             cfg.Webhook.Enable,
-			IsContactMethod:     true,
-			IsEPTarget:          true,
-			IsSchedOnCallNotify: true,
-			DisabledMessage:     "Webhooks must be enabled by an administrator",
+			Type:                  destWebhook,
+			Name:                  "Webhook",
+			Enabled:               cfg.Webhook.Enable,
+			IsContactMethod:       true,
+			IsEPTarget:            true,
+			IsSchedOnCallNotify:   true,
+			SupportsStatusUpdates: true,
```

Comment:
Status updates are currently required for webhooks, we should set it required here to match existing functionality
![image](https://github.com/target/goalert/assets/595010/50d100b2-b791-4747-bed3-21816e11a38d)